versioned_docs/version-0.4.2/tutorials/react-native.mdx

```diff
@@ -124,7 +152,7 @@ Run `pod install` in the /ios directory to install the new pods
 
 ### Native permissions configuration
 
-In order for camera and audio to work you'll need to add some native configuration:
+For this tutorial in order for camera and audio to work you'll need to add some native configuration:
 
 You need to at least set up camera permissions.
```

Comment:
Out of curiosity, is it some limitation of Jellyfish? Can't I just set up the microphone permission?

Reply:
In react native project permissions firstly must be declared in AndroidManifest.xml or info.plist. Then permission must be requested in runtime. Package itself cannot change files in your file structure.